msal4j-sdk/src/integrationtest/java/com.microsoft.aad.msal4j/ClientCredentialsIT.java

```diff
@@ -88,6 +88,25 @@ void acquireTokenClientCredentials_ClientSecret_Ciam() throws Exception {
         assertNotNull(result.accessToken());
     }
 
+    @Test
+    void acquireTokenClientCredentials_Certificate_CiamCud() throws Exception {
```

Comment:
Although this is useful, the more important flow is auth code. Are there any tests for auth code ?

Reply:
In the latest commit I added an integration test for auth code: all CCA, asserts that the authority and authorization endpoints are being set as expected, gets a token with the auth code flow, and does the silent flow to ensure it is cached correctly.